coops-ui/src/mixins/resource-lookup-mixin.ts

```diff
@@ -1,6 +1,7 @@
 import { Component, Vue } from 'vue-property-decorator'
 import { mapState } from 'vuex'
-
+import { AlertMessage } from '@/interfaces'
```

Comment:
I believe we added an `IF` onto the end of our interfaces, ie `AlertMessageIF`. Not sure if you want to just to keep things consistent.